### PR TITLE
update alt text, remove metric from program

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?

alt-text of left nav said 'analyses'
users don't setup program metrics, they set up programs which have metrics

#### What's this PR do?
* update text...

### CODE REVIEW

**Comment by nllong** [comment]
@kflemin / @anchapin  -- any updates with this update? These "issues" were raised by Jay/ACE.

File: seed/templates/seed/_sidebar.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
             <div class="item_name" translate>Organizations</div>
             <div ng-show="logged_in" class="badge badge_menu">{$ organizations_count | number:0 $}</div>
         </a>
-        <a id="sidebar-accounts" ng-class="{item: true, active: is_active('/insights'), 'disabled-item': !logged_in}" title="{$:: 'Analyses' | translate $}" href="{% url "seed:home" %}#/insights">
+        <a id="sidebar-accounts" ng-class="{item: true, active: is_active('/insights'), 'disabled-item': !logged_in}" title="{$:: 'Insights' | translate $}" href="{% url "seed:home" %}#/insights">
```
Comment (nllong) [inline seed/templates/seed/_sidebar.html:32]: this was an actual bug

**Review by kflemin — APPROVED** [review]

**Review by anchapin — COMMENTED** [review]
Program Config page
Should we change this to just "General Settings"? Otherwise everything else looks good.

**Comment by nllong** [comment]
> Program Config page
> Should we change this to just "General Settings"? Otherwise everything else looks good.

nice catch. Thanks @kflemin and @anchapin for the last minute review!

**Pull request merged**